apps/api/src/server/applications/application/documents/document.service.js

```diff
@@ -121,27 +121,16 @@ const mapDocumentsToSendToBlobStorage = (documents, caseReference) => {
  * @returns {Promise<void>}
  */
 const upsertDocumentVersionsMetadataToDatabase = async (
-	documentsToUpload,
 	blobStorageDocuments,
 	blobStorageContainer
 ) => {
-	// Create a map of document names to their metadata objects for easy lookup
-	const documentNameMap = new Map();
-
-	for (const document of blobStorageDocuments) {
-		documentNameMap.set(document.documentName, document);
-	}
-
 	// Generate an array of documents to upsert, with metadata pulled from the blob storage documents
-	const documentsMetadataToSendToDatabase = documentsToUpload.map((documentToUpload) => {
-		// Look up the metadata for the current document by its name
-		const document = documentNameMap.get(documentToUpload.documentName);
-
+	const documentsMetadataToSendToDatabase = blobStorageDocuments.map((documentToUpload) => {
 		// Create an object containing the metadata to upsert for the current document
 		return {
 			blobStorageContainer,
-			documentGuid: document.GUID,
-			documentURI: document.blobStoreUrl
+			documentGuid: documentToUpload.GUID,
+			documentURI: documentToUpload.blobStoreUrl
```

Comment:
This is not going to work since `documentToUpload` is undefined

Reply:
it will because I changed the data is comming from blobStorageDocuments. thats why I changed it

Reply:
AH did not notice is being defined earlier, sorry!